packages/react/src/components/MDXComponents/overview.mdx

```diff
@@ -0,0 +1,64 @@
+<Meta title="MDX Components/MDX Documentation Guidelines" />
+# MDX Documentation Guidelines  
+
+This document outlines the best practices and rules for writing MDX documentation 
+for components under `@carbon-labs`. 
+
+Following these guidelines will ensure all documented components have the same styles
+as the ones found in the [Carbon Design System](carbondesignsystem.com) page.
+
+
+## Directory Structure  
+- All `.mdx` files **must** be placed under:  
+  ```
+  src/components/ComponentName/docs/
+  ```
```

Comment:
I don't think this is true any more. For example, the example component uses a `__docs__` directory. We can remove this. Our MDX gets used if it's an MDX file, and it doesn't have an attached Component Story Format (CSF) like the autodocs MDX have.